pkg/embed/embed_test.go

```diff
@@ -41,15 +41,18 @@ func TestEmbedReleaseDataInBinary(t *testing.T) {
 	gotBinContent, err := os.ReadFile(outputFile.Name())
 	assert.NoError(t, err)
 
-	wantBinContent := append(binContent, []byte(beginReleaseDelimiter)...)
+	beginString := "-----BEGIN APP RELEASE-----"
```

Comment:
why not use the `beginReleaseDelimiterBytes()` and `endReleaseDelimiterBytes()` functions here as well?

Reply:
I'll do that and add unit tests for the bytes functions